curriculum/challenges/english/25-front-end-development/quiz-css-animations/66ed8fc9f45ce3ece4053eae.md

```diff
@@ -17,439 +17,429 @@ Answer all of the questions below correctly to pass the quiz.
-
+Repeats the animation infinitely with no delay
 ---
 
-Placeholder distractor 3
+Fades in and out during each step
 
 #### --answer--
 
-Placeholder answer
+Jumps immediately to the next frame
+
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which property controls the reverse animation direction in this example?
 
 #### --distractors--
 
-Placeholder distractor 1
-
+The animation will loop infinitely
 ---
 
-Placeholder distractor 2
-
+The animation will run forward then reverse
 ---
 
-Placeholder distractor 3
+The animation will reset after each iteration
 
 #### --answer--
 
-Placeholder answer
+The animation will run backward
 
 ### --question--
 
 #### --text--
 
-Placeholder question
-
+What will happen when this animation is applied?
 #### --distractors--
 
-Placeholder distractor 1
+ The element will move left
 
 ---
 
-Placeholder distractor 2
+The element will move 200px to the bottom
 
 ---
 
-Placeholder distractor 3
+The element will rotate 200 degrees
 
 #### --answer--
 
-Placeholder answer
+The element will move 200px to the right
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How will the animation behave if animation-iteration-count is set to infinite?
 
 #### --distractors--
 
-Placeholder distractor 1
+The animation will run once and stop
 
 ---
 
-Placeholder distractor 2
+The animation will pause after the first iteration
 
 ---
 
-Placeholder distractor 3
+The animation will stop after 3 iterations
 
 #### --answer--
 
-Placeholder answer
+The animation will repeat infinitely
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which keyframe selector defines the halfway point of an animation?
 
 #### --distractors--
 
-Placeholder distractor 1
+0%
 
 ---
 
-Placeholder distractor 2
+25%
 
 ---
 
-Placeholder distractor 3
+100%
 
 #### --answer--
 
-Placeholder answer
+50%
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How do you ensure an animation starts from the middle of its defined steps?
 
 #### --distractors--
 
-Placeholder distractor 1
+reverse
 
 ---
 
-Placeholder distractor 2
+forwards
+
 
 ---
 
-Placeholder distractor 3
+paused
 
 #### --answer--
 
-Placeholder answer
+backwards
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the effect of using the ease-in-out timing function?
 
 #### --distractors--
 
-Placeholder distractor 1
+The animation will start fast and end slow
 
 ---
 
-Placeholder distractor 2
+The animation will run at a constant speed
 
 ---
 
-Placeholder distractor 3
+The animation will loop infinitely
 
 #### --answer--
 
-Placeholder answer
+The animation will start slow, speed up, then slow down
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which property prevents the animation from running immediately?
 
 #### --distractors--
 
-Placeholder distractor 1
+animation-fill-mode
 
 ---
 
-Placeholder distractor 2
+animation-timing-function
 
 ---
 
-Placeholder distractor 3
+animation-iteration-count
```

Comment:
```suggestion
`animation-iteration-count`
```

Reply:
all comments are reslove please check update @naomi-lgbt

Reply:
@rachhanari they aren't, for example you did not add the backticks like requested in this comment